src/api/exceptions/ModelNotFoundException.ts

```diff
@@ -3,7 +3,7 @@
 // file COPYING or https://github.com/particl/particl-market/blob/develop/LICENSE
 
 /**
- * MessageException
+ * ModelNotFoundException
  * ----------------------------------------
  *
  * This should be used if a someone requests a
```

Comment:
This should be used if the command that was called, was given a parameter, which then was used to fetch some other model which doesnt exists..